go/mysql/server.go

```diff
@@ -129,17 +147,22 @@ func (l *Listener) Accept() {
 			return
 		}
 
+		acceptTime := time.Now()
+
 		connectionID := l.connectionID
 		l.connectionID++
 
-		go l.handle(conn, connectionID)
+		connCount.Add(1)
+		connAccept.Add(1)
+
+		go l.handle(conn, connectionID, acceptTime)
 	}
 }
 
 // handle is called in a go routine for each client connection.
 // FIXME(alainjobart) handle per-connection logs in a way that makes sense.
 // FIXME(alainjobart) add an idle timeout for the connection.
-func (l *Listener) handle(conn net.Conn, connectionID uint32) {
+func (l *Listener) handle(conn net.Conn, connectionID uint32, acceptTime time.Time) {
 	c := newConn(conn)
```

Comment:
acceptTime could be initialized here, at the beginning of this handle() method, so you don't have to pass it in. It wouldn't be 100% accurate (time to launch a go routine would be subtracted) but would be simpler. Up to you.